lib/core/src/main/java/io/atlasmap/actions/CollectionActions.java

```diff
@@ -0,0 +1,17 @@
+package io.atlasmap.actions;
```

Comment:
Now I wonder if we need this class - since actual transformation entry in the mapping definition is removed in DefaultAtlasContext. Maybe just the `CopyTo` model class is enough?